core/src/main/resources/reference.conf

```diff
@@ -62,6 +62,11 @@ akka.kafka.consumer {
   # After exceeding maxinum wakeups the consumer will stop and the stage will fail.
   max-wakeups = 10
 
+  # If enabled the consumer will re-send last committed offsets periodically
+  # for all assigned partitions. See https://issues.apache.org/jira/browse/KAFKA-4682.
+  commit-refresh-enabled = false
+  commit-refresh-interval = 1m
```

Comment:
would it not be nicer to have one value, and it being `infinite` being "disabled"? That's usually how we have such settings in Akka and Akka HTTP

Reply:
It is indeed much nicer, but I didn't want to introduce a custom config behavior. As you say it is already done like this in akka, I will change the code here to use a single config value too.